destiny/main/bdd/models/bans.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy import Column, ForeignKey
+from sqlalchemy.sql.sqltypes import Boolean, BigInteger, Integer, CHAR, SmallInteger
+
+from destiny.main.bdd import Base
+from destiny.utils import rep_model
+
+
+class Bans(Base):
+    __tablename__ = 'bans'
+
+    gameId = Column(BigInteger, ForeignKey("teamstats.gameId"), primary_key=True)
+    teamId = Column(SmallInteger, primary_key=True)
+    pickTurn = Column(SmallInteger)
+    championId = Column(SmallInteger, primary_key=True)
+
```

Comment:
I write it once but it refers to all classes:
Sqlalchemy allows to build relationships between model (via foreignkey) objects and this is one of the most powerfull feature of sqlalchemy:
http://docs.sqlalchemy.org/en/latest/orm/relationships.html

I think it should be implemented asap because sqlalchemy without this feature is not that usefull..